decosjoin/api/decosjoin/decosjoin_connection.py

```diff
@@ -167,14 +167,20 @@ def _transform(self, zaken, user_identifier):
 
             elif f['text45'] == 'B&B - vergunning':
                 fields = [
-                    {"name": "caseType", "from": 'text45', "parser": to_string},
+                    {"name": "caseType", "from": "text45", "parser": to_string},
                     {"name": "dateRequest", "from": "document_date", "parser": to_date},  # Startdatum zaak
-                    {"name": "location", "from": 'text6', "parser": to_string},
-                    {"name": "identifier", "from": 'mark', "parser": to_string},
-                    {"name": "title", "from": 'subject1', "parser": to_string},
-                    {"name": "status", "from": 'title', "parser": to_string},
-                    {"name": "requester", "from": 'company', "parser": to_string},
-                    {"name": "dateStart", "from": "document_date", "parser": to_date},  # Startdatum zaak
+                    {"name": "result", "from": "dfunction", "parser": to_string},
+                    {"name": "location", "from": "text6", "parser": to_string},
+                    {"name": "title", "from": "subject1", "parser": to_string},
+                    {"name": "identifier", "from": "mark", "parser": to_string},
+                    {"name": "processed", "from": "processed", "parser": to_string},
+                    {"name": "dateProcessed", "from": "date5", "parser": to_datetime},  # Datum afhandeling
+                    {"name": "dateStart", "from": 'date6', "parser": to_date},  # Datum van
+                    {"name": "dateEnd", "from": 'date7', "parser": to_date},  # Datum tot
+                    {"name": "status", "from": "title", "parser": to_string},
+                    {"name": "requester", "from": "company", "parser": to_string},
+                    {"name": "owner", "from": "text25", "parser": to_string},
+                    {"name": "before2019", "from": "text20", "parser": to_string},  # Todo: is this a date?
```

Comment:
Wat kunnen we hiermee?

Reply:
Goede vraag, stond in het RD van Micha. Ik heb nog geen logica gevonden die dit gebruikt.